docs/ubuntu-24.md

```diff
@@ -0,0 +1,61 @@
+## Installing on Ubuntu 24.04
+
+Ubuntu 24.04 [introduced restrictions](https://discourse.ubuntu.com/t/ubuntu-24-04-lts-noble-numbat-release-notes/39890) for unpriviledged user namespaces that prevents electron based apps from running without an appropriate apparmor profile.
+
+## Solution 1
+
+Install launcher using the .deb file. This will create an appropriate apparmor profile under the hood for you and you don't need to bother about it.
```

Comment:
Can someone check whether the deb file is indeed working now without further actions? [This](https://github.com/holochain/launcher-electron/releases/download/v0.300.0/holochain-launcher-0.3_0.300.0_amd64.deb) deb file may be used to test it.

Reply:
The deb produces a broken install for me

```
LaunchProcess: failed to execvp:
/opt/Holochain
[592558:0717/142300.319452:FATAL:zygote_host_impl_linux.cc(202)] Check failed: . : Invalid argument (22)
Trace/breakpoint trap (core dumped)
```

Reply:
Okay, thanks for testing. I might have missed another set of quotes somewhere.